packages/lambda/src/api/render-media-on-lambda.ts

```diff
@@ -111,6 +112,13 @@ export const renderMediaOnLambda = async ({
 		durationInFrames: 1,
 	});
 	validateDownloadBehavior(downloadBehavior);
+	if (outName === undefined || typeof outName === 'string') {
+		validateOutputFilename(
+			codec,
+			RenderInternals.getExtensionOfFilename(outName ?? null)
+		);
+	}
+
```

Comment:
@JonnyBurger is there a way to use this function without importing `@remotion/renderer`. It causes tests to fail.